app/core/utils.js

```diff
@@ -209,10 +209,9 @@ utils.processPatientData = (data, queryParams, timePrefs, bgPrefs) => {
   }
   // but otherwise we use the timezone from the most recent upload metadata obj
   else {
-    // @todo: need to figure out a way to refactor this bit of shenanigans
-    // comp.setState({
-    //   timePrefs: timePrefsForTideline
-    // });
```

Comment:
Since you removed the TODO, no longer thinking about refactoring? I don't think I mind if you don't - since this preprocessing step is one of the things on the chopping block the hackery is not _too_ bad.